pkgs/by-name/ha/hare/package.nix

```diff
@@ -23,8 +23,7 @@ assert
     inherit (lib) intersectLists platforms concatStringsSep;
     workingPlatforms = intersectLists platforms.linux (with platforms; x86_64 ++ aarch64 ++ riscv64);
   in
-  (enableCrossCompilation -> !(isLinux && is64bit))
-  -> builtins.throw ''
+  (enableCrossCompilation -> !(isLinux && is64bit)) ''
```

Comment:
> error: attempt to call something which is not a function but a Boolean: false

This is being interpreted as applying the string to the boolean condition.

I'm guessing the `assert` keyword needs to be moved after the `let in` block, so that the boolean is applied to `assert` instead.